examples/workdir/env/extravars

```diff
@@ -19,6 +19,12 @@ redeploy_osh_only: false
 #suse_osh_registry_location: "docker.io"
 #suse_osh_image_version: "pike"
 
+# For TLS CA cert: common name used by default
+#suse_tls_cert_CA_common_name: "ca.cluster.local"
+
+# For TLS cert: common name used by default
+#suse_tls_cert_common_name: "server.cluster.local"
```

Comment:
we should allow user defined domain suffix. but that can be in a different commit.